UC4-chaincode/src/test/java/de/upb/cs/uc4/chaincode/ExamContractTest.java

```diff
@@ -40,6 +41,8 @@ DynamicTest CreateTest(JsonIOTest test) {
                 return DynamicTest.dynamicTest(testName, addExamFailureTest(setup, input, compare, ids));
             case "getExam_SUCCESS":
                 return DynamicTest.dynamicTest(testName, getExamsSuccessTest(setup, input, compare, ids));
+            case "getExam_FAILURE":
+                return DynamicTest.dynamicTest(testName, getExamsFailureTest(setup, input, compare, ids));
```

Comment:
```suggestion
```
This is not needed, as the test structure for successful and unsuccessful get requests is the same.